uber/templates/signups/login.html

```diff
@@ -19,15 +19,15 @@ <h2 class="form-signin-heading">Volunteer Login</h2>
         </form>
     {% else %}
         <div class="well">
-            Shifts are not yet available. If you are signed up as a volunteer, you will 
+            Shifts are not yet available. If you are signed up as a volunteer, you will
```

Comment:
Haha, it just looked like it might have been accidental/overlooked. I suppose a linebreak will also render as a space? It should be easy enough to take a quick look at the page before merging.